content/applications/services/helpdesk/overview/sla.rst

```diff
@@ -2,183 +2,209 @@
+report and corresponding pivot table by going to :menuselection:`Helpdesk app --> Reporting --> SLA
+Status Analysis`.
 
 Using the Pivot view
 --------------------
 
-By default, the report is displayed in a :guilabel:`Pivot` view, and is filtered to show the number
-of SLAs failed and the failure rate over the last 30 days, grouped by team.
+By default, the report displays in a :guilabel:`Pivot` view. Any :abbr:`SLA (Service Level
+Agreement)` policies in the database with tickets that failed to fulfill a policy, are in progress,
+or have satisfied a policy are listed. By default, they are grouped by team and ticket count.
 
 .. image:: sla/sla-status-analysis.png
    :align: center
-   :alt: View of the SLA status analysis report in Odoo Helpdesk
+   :alt: View of the SLA status analysis report in Odoo Helpdesk.
+
+To change the display, or to add additional measurements, click the :guilabel:`Measures` button to
+reveal a drop-down menu of reporting criteria, and choose from the options available. Whenever a
+measurement is picked, a check mark appears in the drop-down menu to indicate that the measurement
+is included, and a corresponding new column will emerge in the pivot table to show the relevant
+calculations.
+
+.. image:: sla/sla-pivot-measures.png
+   :align: center
+   :alt: View of the available measures in the SLA status analysis report.
 
-To add the number of SLAs passed or in progress, click the :guilabel:`Measures` button to reveal a
-drop-down menu of reporting criteria, and choose from the options available based on the
-measurements preferred. Whenever a measurement is picked, a check mark will appear in the drop-down
-menu to indicate that that measurement is included, and a corresponding new column will emerge in
-the pivot table to show the relevant calculations.
+To add a group to a row or column, click the :guilabel:`➕ (plus)` icon next to the policy name and
+then select one of the groups. To remove one, click the :guilabel:`➖ (minus)` icon next to the
+policy name.
 
-To add a group to a row or column, click the plus :guilabel:` + ` button next to
-:guilabel:`Total`, and then select one of the groups. To remove one, click the minus :guilabel:` - `
-button and deselect.
+.. image:: sla/sla-pivot-groups.png
+   :align: center
+   :alt: View of the available group by options in the SLA status analysis report.
 
 Using the Graph view
 --------------------
 
-The :guilabel:`Status Analysis` report can also be viewed as a :guilabel:`Bar`, :guilabel:`Line`, or
-:guilabel:`Pie` Chart. Toggle between these views by selecting the appropriate icon at the top of
-the chart.
+The :guilabel:`SLA Status Analysis` report can also be viewed as a :guilabel:`Bar`,
+:guilabel:`Line`, or :guilabel:`Pie` Chart. Toggle between these views by first selecting the
+:guilabel:`Graph` icon at the top-right of the dashboard. Then select the appropriate chart icon at
+the top-left of the graph.
```

Comment:
```suggestion
The :guilabel:`SLA Status Analysis` report can also be viewed as a :guilabel:`Bar Chart`,
:guilabel:`Line Chart`, or :guilabel:`Pie Chart`. Toggle between these views by first selecting the
:guilabel:`Graph` button at the top-right of the dashboard. Then, select the appropriate chart icon
at the top-left of the graph.
```

Reply:
added the chart to the end of those graph options because that's the name of the button when the cursor hovers over it. I applied that same methodology to the Graph button, as well.